Stratis.Bitcoin/Base/ChainState.cs

```diff
@@ -80,5 +80,13 @@ public ChainedBlock HighestPersistedBlock
         {
             get; set;
         }
+
+        /// <summary>
+        /// Represents the last block stored to the index repository
+        /// </summary>
+        public ChainedBlock HighestIndexedBlock
```

Comment:
The difference is BlockStore is a required feature to run a fullnode but Index store is not crucial, that's why the separation.